newsroom/products/views.py

```diff
@@ -44,61 +39,56 @@ def get_product_ref(product: Product, seats=0) -> ProductRef:
     }
 
 
-@blueprint.route("/products", methods=["GET"])
-@admin_only
+class SearchParams(BaseModel):
+    q: str | None = None
```

Comment:
It would be good to consider updating the variable name `q` to something more descriptive, such as `query`. thoughts?

Reply:
I think this parameter is coming as `q` from the frontend so I wouldn't change it for now to reduce the code changes to the frontend.